### DISCUSSION TITLE
How to get exrc to play nicely with ftplugin

### DISCUSSION THREAD

**Discussion by dbatten5** [neovim/neovim#27267] [Q&A]
I'm using `exrc` mode to set some project local settings. These settings get overriden if a `ftplugin` file is set for the current filetype. My usecase is that I've set a colorcolumn option in `ftplugin/python.lua` and I want to override that for a particular project. How can I go about that?

**Comment by ishaan-kapoor** [comment]
hey, could you find a solution?

**Comment by Phantasimay** [comment]
you can make ftplugin to respect existence of `.exrc` with:

```viml
if filereadable(".exrc")
	finish
endif
```
on top of `ftplugin/python.vim` so it will prevent to source script , or in `ftplugin/python.lua` 

```lua
local M={}

M.Your_config()
          bal bla bla
          bla bla bla
end

if vim.fn.filereadable(".exrc") == 1 then
	return {}
else
        return M
end
```
